doc/neps/nep-0000.rst

```diff
@@ -0,0 +1,219 @@
+
+Each NEP must have a champion---someone who writes the NEP using the style
+and format described below, shepherds the discussions in the appropriate
+forums, and attempts to build community consensus around the idea.  The NEP
+champion (a.k.a. Author) should first attempt to ascertain whether the idea is
+NEP-able. Posting to the numpy-discussion `mailing list`_ is the best way to go
+about doing this.
+
+Following a discussion on the mailing list, the proposal should be submitted as a
+draft NEP via a `GitHub pull request`_ to the ``doc/neps`` directory with the name
+``nep-<n>.rst`` where ``<n>`` is an appropriately assigned number. The draft must
+use the :doc:`nep-template` file. Once a formal proposal has been submitted as a
+PR, it should be announced on the mailing list.
+
+Standards Track NEPs consist of two parts, a design document and a
+reference implementation.  It is generally recommended that at least a
+prototype implementation be co-developed with the NEP, as ideas that sound
+good in principle sometimes turn out to be impractical when subjected to the
+test of implementation.
+
+
+Review and Resolution
+^^^^^^^^^^^^^^^^^^^^^
+
+NEPs are discussed on the mailing list and perhaps in other forums.
+Sometimes NEPs will grow out of an existing pull request.
+The possible paths of the status of NEPs are as follows:
+
+.. image:: _static/nep-0000.png
+
+All NEPs should be created with the ``Draft`` status.
+
+Normally, a NEP is ``Accepted`` by consensus of all
+interested Contributors.
+In unusual cases, the `NumPy Steering Council`_ may be asked to decide whether
+a controversial NEP is ``Accepted``.
+
+Once a NEP has been ``Accepted``, the reference implementation must be
+completed.  When the reference implementation is complete and incorporated
+into the main source code repository, the status will be changed to ``Final``.
+
+A NEP can also be assigned status ``Deferred``.  The NEP author or a
+core developer can assign the NEP this status when no progress is being made
+on the NEP.
+
+A NEP can also be ``Rejected``.  Perhaps after all is said and done it
+was not a good idea.  It is still important to have a record of this
+fact. The ``Withdrawn`` status is similar---it means that the NEP author
+themselves has decided that the NEP is actually a bad idea, or has
+accepted that a competing proposal is a better alternative.
+
+When a NEP is ``Accepted``, ``Rejected``, or ``Withdrawn``, the NEP should be
+updated accordingly. In addition to updating the status field, at the very
+least the ``Resolution`` header should be added with a link to the relevant
+post in the mailing list archives.
+
+NEPs can also be ``Replaced`` by a different NEP, rendering the original
+obsolete.  Some Informational and Process NEPs may also have a status of
+``Active`` if they are never meant to be completed.  E.g. NEP 0 (this NEP).
+
+
+Maintenance
+^^^^^^^^^^^
+
+In general, Standards track NEPs are no longer modified after they have
```

Comment:
Clarify language.

Reply:
I added some additional text.  Feel free to suggest further changes.